### PR TITLE
Update AboutUs.md

### LINKED ISSUE #35: Update AboutUs and README

### PR DESCRIPTION
**Pull request by Whleee**
Portfolio page still default, will be updated in the future.

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -1,59 +1,51 @@
...
-### John Doe
+### Faqih
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/abdurrahmanfaqihiskandar.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/abdurrahmanfaqihiskandar)][[portfolio](team/johndoe.md)]
```
Comment (faqihisaninja) [inline docs/AboutUs.md]: I think can remove all the portfolio section. Since it's all johndoe.md

**Review by faqihisaninja — APPROVED** [review]
Other than the comment I have, it looks good. Thanks Wei Heng!

File: README.md [hunk 2]
```diff
@@ -14,4 +14,132 @@ Some of the features include:
...
 For more in-depth information on the CanoE-COACH visit [our website](https://ay2021s1-cs2103-f10-1.github.io/tp/).
 
+1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+
+1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.
+
+1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+   Some example commands you can try:
+
+   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
```
Comment (yejiadong) [inline README.md:24]: Might want to fetch the updated UserGuide.md to paste here, as it was edited and merged yesterday to resolve some docu bugs (e.g. address here is removed)

File: docs/AboutUs.md [hunk 3]
```diff
@@ -1,59 +1,51 @@
----
-layout: page
-title: About Us
----
+# About Us
 
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
 You can reach us at the email `seer[at]comp.nus.edu.sg`
```
Comment (yejiadong) [inline docs/AboutUs.md]: Think can change the email here to one of our email addresses

**Review by yejiadong — COMMENTED** [review]
LGTM, with afew nits to fix.

File: docs/AboutUs.md (hunk 3 above)
Comment (Whleee) [reply docs/AboutUs.md]: sure I just put mine

File: README.md (hunk 2 above)
Comment (Whleee) [reply README.md:24]: yup done

File: docs/AboutUs.md (hunk 1 above)
Comment (Whleee) [reply docs/AboutUs.md]: yup done

**Review by yejiadong — APPROVED** [review]

**Pull request merged**